deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/layers/HelperUtils.java

```diff
@@ -33,6 +33,9 @@
 @Slf4j
 public class HelperUtils {
 
+    public static final String DISABLE_HELPER_PROPERTY = "org.eclipse.deeplearning4j.helpers.disable";
+    public final static String HELPER_DISABLE_DEFAULT_VALUE = "true";
```

Comment:
We've got https://github.com/eclipse/deeplearning4j/blob/master/nd4j/nd4j-common/src/main/java/org/nd4j/common/config/ND4JSystemProperties.java and https://github.com/eclipse/deeplearning4j/blob/master/deeplearning4j/deeplearning4j-common/src/main/java/org/deeplearning4j/common/config/DL4JSystemProperties.java as a place to define constants (and docs) for System Properties. Please put it there.